tests/sentry/incidents/endpoints/test_organization_alert_rule_anomalies.py

```diff
@@ -0,0 +1,153 @@
+from datetime import timedelta
+from unittest.mock import patch
+
+# import orjson
+import pytest
+from urllib3.response import HTTPResponse
+
+# from sentry.conf.server import SEER_ANOMALY_DETECTION_ENDPOINT_URL
+from sentry.incidents.models.alert_rule import (
+    AlertRuleDetectionType,
+    AlertRuleSeasonality,
+    AlertRuleSensitivity,
+)
+from sentry.seer.anomaly_detection.types import AnomalyType
+from sentry.testutils.cases import SnubaTestCase
+from sentry.testutils.factories import EventType
+from sentry.testutils.helpers.datetime import before_now, freeze_time, iso_format
+from sentry.testutils.helpers.features import with_feature
+from sentry.testutils.outbox import outbox_runner
+from sentry.testutils.skips import requires_snuba
+from tests.sentry.incidents.endpoints.test_organization_alert_rule_index import AlertRuleBase
+
+pytestmark = [pytest.mark.sentry_metrics, requires_snuba]
+
+
+@freeze_time()
+class AlertRuleAnomalyEndpointTest(AlertRuleBase, SnubaTestCase):
+    endpoint = "sentry-api-0-organization-alert-rule-anomalies"
+
+    method = "get"
+
+    @with_feature("organizations:anomaly-detection-alerts")
+    @with_feature("organizations:incidents")
+    @patch(
+        "sentry.seer.anomaly_detection.store_data.seer_anomaly_detection_connection_pool.urlopen"
+    )
+    # @patch(
```

Comment:
is all the commented out stuff for the next pr?

Reply:
Yes it is, I figured since I'm planning to merge the next one in quick succession I would leave the comments in but I'll take them out for better style

Reply:
:nod:

Yea - ferge asked us to split the previous PR up 😅 

probably not necessary since we have context, but i suppose its good practice lol